src/shared/clients/codecatalystClient.ts

```diff
@@ -320,6 +320,7 @@ class CodeCatalystClientInternal {
         const resp = await this.call(this.sdkClient.verifySession(), false)
         assertHasProps(resp, 'identity')
 
+        CodeCatalystClientInternal.identityCache.set(accessToken, resp.identity)
```

Comment:
do we need a source comment here, this seems kind of special and could get lost or at least the purpose isn't obvious

Reply:
Yeah I'll add some notes